app/src/main/java/ch/hikemate/app/ui/auth/SignInScreen.kt

```diff
@@ -83,7 +86,7 @@ fun SignInScreen() {
               )
             }
 
-            SignInWithGoogleButton { /* Handle sign in click here */}
+            SignInWithGoogleButton { navigaionActions.navigateTo(TopLevelDestinations.MAP) }
```

Comment:
Warning: Perhaps include a hint here that currently the real sign-in is bypassed and the login screen opened automatically without any checks. Currently it may be misleading.